spec/features/tasks/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'Viewing all tasks' do
+    scenario 'when visiting the page the first time, View All link is available' do
+        visit tasks_path
+        expect(page).to have_content("View All")
+    end
+
+    scenario 'when clicking View All, Incomplete tasks link is available' do
+        visit tasks_path
+        click_on 'View All'
```

Comment:
One thing we are currently brainstorming at Contently is how to speed up our test suite, can you think of a way to have this spec file run faster?

Reply:
we could start the test at the page where the "Incomplete tasks" link would already be available.

Reply:
Nice, I think thats a good idea!